samples/ODataAuthorizationSample/README.md

```diff
@@ -0,0 +1,24 @@
+# OData WebApi Authorization sample
+
+This application demonstrates how to use the OData authorization extensions to apply permissions to OData endpoints based on the model capability restrictions.
+
+The applicaiton defines a model with CRUD permission restrictions annotations on the `Customers` entity set and the
+`GetTopCustomer` unbound function.
+
+It uses a custom authentication handler that assumes a
+user is always authenticated. This handler extracts the permissions from a header called `Permissions`, which
+is a comma-separated list of allowed scopes.
+
+Based on the model annotations, the:
+
+| Endpoint                 | Required permissions
+---------------------------|----------------------
+`GET /odata/Customers`     | `Customers.Read`
+`GET /odata/Customers/1`   | `Customers.ReadByKey`
```

Comment:
Can `Customer.Read` also be defined on this endpoint and if so, would it work just the same as a `ReadByKey`?

Reply:
Technically, it `Customers.Read` applies to the `GET /odata/Customers/1` as well. So if you don't have `ReadByKey`, it will check whether you have `Read` in this case.

Reply:
I've updated the doc to reflect this.